service/surf/apps/filters/utils.py

```diff
@@ -144,7 +150,9 @@ def _fetch_edustandaard_translations(external_id):
 
     dutch_value = json[key]['http://www.w3.org/2004/02/skos/core#prefLabel'][0]['value']
 
-    return dutch_value, dutch_value
+    english_value = _translate_with_deepl(dutch_value)
```

Comment:
Euhm. Blijkbaar is dat lastig refactoren. Want als je dat hier doet krijg je een conflict.

Reply:
wat bedoel je?